packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -821,7 +821,31 @@ export function createLodestarMetrics(
         name: "validator_monitor_validators",
         help: "Count of validators that are specifically monitored by this beacon node",
       }),
-
+      validatorsActive: register.gauge({
+        name: "validator_monitor_validators_active",
+        help: "Count of validators monitored by this beacon node that are active",
+      }),
```

Comment:
All of this should be the same metric with label: "status" where status can have values: "active", "pending", etc

Reply:
I had originally [done something similar](https://github.com/ChainSafe/lodestar/pull/5239#discussion_r1127489060) but it goes against [the spec](https://docs.google.com/document/d/1qPWAVRjPCENlyAjUBwGkHMvz9qLdd_6u9DPZcNxDBpc/edit?usp=drivesdk) , but I agree with you in that this would be more simplified programmatically.